larray_editor/commands.py

```diff
@@ -68,3 +68,45 @@ def redo(self):
             command = "{}[{}] = {}".format(self.array_name, change.key, change.new_value)
             self.editor.kernel.shell.run_cell(command)
         self.editor.arraywidget.model_data.reset()
+
+
+class EditCurrentArrayCommand(QUndoCommand):
```

Comment:
Wouldn't be better to have one common abstract class to inherit from in EditArrayCommand and EditCurrentArrayCommand?